### ISSUE TITLE
Is it possible to get x, y coordinates of detected object and how ? 

### ISSUE THREAD

**Issue by boldyrek** [TensorBox/TensorBox#220]

**Comment by Russell91** [comment]
Yes, the predictions come in a 15x20 grid. For each cell in the grid, you get a confidence in [0,1], and 4 numbers, corresponding to center_y, center_x, height, and width, respectively.

**Comment by RomanSteinberg** [comment]
Explore `evaluate.py`. Or you can use `predict.py` from my [fork](https://github.com/RomanSteinberg/TensorBox). Hope it will be merged.
